apps/remote_control/lib/lexical/remote_control/search/indexer/entry.ex

```diff
@@ -0,0 +1,90 @@
+defmodule Lexical.RemoteControl.Search.Indexer.Entry do
+  @type entry_type :: :module | :function | :macro | :field
+  @type subject :: String.t()
+  @type entry_subtype :: :reference | :definition
+  @type indexer :: :beam | :source
+  @type line :: non_neg_integer()
+  @type column :: non_neg_integer
+  @type position :: {line, column}
+  @type version :: String.t()
+  @type entry_reference :: reference() | nil
+
+  defstruct [
+    :application,
+    :elixir_version,
+    :erlang_version,
+    :finish,
```

Comment:
Is this the end position? Can we call it `:end` instead for consistency with ranges?

Reply:
I really hate calling things 'end' as it conflicts with elixir's syntax for `end` and throws my editor into knots, though I might just use `Lexical.Document.Range` to represent them. I think that's the better idea.